mne/io/brainvision/brainvision.py

```diff
@@ -296,9 +298,18 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
                       "not the '%s' file." % ext)
     with open(vhdr_fname, 'rb') as f:
         # extract the first section to resemble a cfg
-        header = f.readline().decode('utf-8').strip()
+        header = f.readline()
+        try:
+            header = header.decode('utf-8').strip()
+        except UnicodeDecodeError:
+            header = header.decode('ISO-8859-1')
```

Comment:
If I'm not mistaken, those are synonyms ... It seems the newer files use 'utf-8' by default while the older ones use the other one (which is the default encoding for non-ascii text files on Windows and MS Office still tries to impose). There is a line in the newer files specifying codepage, but it is a bit more effort to actually parse that. Should still be doable though.